sort-python-dictionary/compare_sorting_dict_vs_list.py

```diff
@@ -0,0 +1,29 @@
+from timeit import timeit
+
+list_program = "sorted(list_of_dictionaries, key=lambda item:item['age'])"
+dict_program = """
```

Comment:
Your variables are named differently than in the tutorial, and there's also a difference in how you import the samples. The code formatting seems to be different, too.

Reply:
I've made this more consistent